com.civicrm.hrjobroles/hrjobroles.php

```diff
@@ -135,14 +135,19 @@ function hrjobroles_civicrm_navigationMenu( &$params ) {
  * @param $tabs
  * @param $contactID
  * Create a custom tab for civicrm contact which will implement custom drupal callback function
+ * this tab should appear after job contracts tab directly
+ * and since contact summary tab weight is
+ * -190 we choosed this to be -180
```

Comment:
Everything look much better now with the comments! Just one minor problem: Choose is an irregular verb, so the past tense here would be chose